include/jets.hxx

```diff
@@ -4,6 +4,21 @@
 namespace physicsobject {
 namespace jet {
 
+
+ROOT::RDF::RNode 
+JetID(ROOT::RDF::RNode df,
+              correctionManager::CorrectionManager &correction_manager,
+              const std::string &outputname,
+              const std::string &jet_eta,
+              const std::string &jet_chHEF,
+              const std::string &jet_neHEF,
+              const std::string &jet_chEmEF,
+              const std::string &jet_neEmEF,
+              const std::string &jet_muEF,
+              const std::string &jet_chMult,
+              const std::string &jet_neMult,
+              const std::string &jet_id_file,
+              const std::string &jet_name);
```

Comment:
I suggest to change the namespace of this function. Since this is a jet quantity, I would move it to: `physicsobject::jet::quantity::ID` with the function name `ID`. Similar to the tau IDFlag quantity.